src/wallet/wallet.js

```diff
@@ -15,26 +15,32 @@ export class Wallet {
    * @constructor
    *
    * @param {string} email User's email.
-   * @param {Keypair|string} keypair User's key pair or a secret seed.
+   * @param {Keypair|string} signingKeypair User's key pair or a secret seed.
    * @param {string} accountId User's account ID.
    * @param {string} [walletId] Wallet ID.
    * @param {string} [sessionId] Session ID.
    * @param {string} [sessionKey] Session key.
+   * @param {Array} [keypairs] array of {@link Keypair} or strings(secret seed) which saved in key storage
    */
-  constructor (email, keypair, accountId, walletId, sessionId, sessionKey) {
+  constructor (email, signingKeypair, accountId, walletId, sessionId, sessionKey, keypairs) {
     if (isNil(email)) {
       throw new Error('Email is required.')
     }
 
-    if (isNil(keypair)) {
-      throw new Error('No keypair provided.')
-    } else if (isString(keypair)) {
-      if (!Keypair.isValidSecretKey(keypair)) {
-        throw new Error('Invalid secret seed.')
-      }
-      keypair = Keypair.fromSecret(keypair)
-    } else if (!(keypair instanceof Keypair)) {
-      throw new Error('Invalid keypair. Expected a Keypair instance or a string seed.')
+    if (!Array.isArray(keypairs)) {
```

Comment:
if I occasionally pass there an object/string/number - it will be hard to locate a bug, because it will be just replaced with an empty array.

I'd rather make a default value for keypairs (`keypairs = []`) or just make a check for it's presense: 
`if (keypairs) { keypairs.map(...) }`